karate-core/src/main/java/com/intuit/karate/core/ScenarioContext.java

```diff
@@ -1149,4 +1151,28 @@ public void stop(StepResult lastStepResult) {
         }
     }
 
+    public void lightHouseWrapperAction(String URL, List<String> options) throws IOException, InterruptedException {
```

Comment:
actual lighthouse wrapper implementation - this can be configured further for many different options.

Reply:
so it IS a CLI - ! see my comment above. I think you should first get that working as a custom feature and experiment - that may be a good enough first version / experiment

Reply:
yeah! actually, in that case we might not need this at all - since Karate CLI support is good enough to run this all -

    * def proc = karate.fork('lighthouse https://github.com/login --only-categories=accessibility,performance -- output=json --emulated-form-factor=none --throttling-method=provided')
    * proc.waitSync()
    * match proc.exitCode == 0

and I hope there is support to parse the response from command to fetch scores / details for categories like accessibility / performance etc.

Reply:
ok, so let us start with this simple approach then. this is a good starting point to invite more contributions - at least it helps others including me to get up and running with axe / lighthouse

Reply:
ok, so anything further needed for this work? since its all pre-existing with CLI support...

Reply:
yes I think for now this PR can be parked. instead I think if you create a Git example project maybe with a Docker script included that shows how you can call axe / lighthouse and integrate into a karate test via CLI - that would be a great reference that we can build on. does that make sense ?

Reply:
yeah sure, will do. Thank you.